planning/behavior_path_planner/src/scene_module/pull_over/pull_over_module.cpp

```diff
@@ -83,6 +83,10 @@ void PullOverModule::resetStatus()
 // This function is needed for waiting for planner_data_
 void PullOverModule::updateOccupancyGrid()
 {
+  if (planner_data_->occupancy_grid == nullptr) {
```

Comment:
I would like this expression like other modules how about this?
https://github.com/autowarefoundation/autoware.universe/blob/34bf3baff102c617a3bc54058fdc97216be8131b/planning/behavior_velocity_planner/src/node.cpp#L224
https://github.com/autowarefoundation/autoware.universe/blob/34bf3baff102c617a3bc54058fdc97216be8131b/planning/behavior_path_planner/src/behavior_path_planner_node.cpp#L515
```suggestion
  if (!planner_data_->occupancy_grid) {
```